pkgs/applications/networking/mumble/default.nix

```diff
@@ -66,6 +66,7 @@ let
         cmakeFlags = [
           "-D g15=OFF"
           "-D CMAKE_CXX_STANDARD=17" # protobuf >22 requires C++ 17
+          "-D CMAKE_UNITY_BUILD=ON" # Upstream uses this in their build pipeline to speed up builds
```

Comment:
But what does this option do?

Reply:
I am no C++ expert, but to my understanding this combines several c++ source or header files into bigger compilation units to speed up compilation. I mainly wanted it to make sure that we compile the code as similar to upstream as possible. [Also see the CMAKE Documentation on that feature](https://cmake.org/cmake/help/v3.31/variable/CMAKE_UNITY_BUILD.html)